src/main/java/com/flowingcode/vaadin/addons/chipfield/ChipField.java

```diff
@@ -302,6 +302,7 @@ public void setNewItemHandler(SerializableFunction<String, T> handler) {
 	}
 
 	@Override
+	@SuppressWarnings("squid:S1186")

```

Comment:
It's not clear why the method does not have a method body: is it an unintentional omission? is it not supported? is it an intentionally-blank override? Remove the `@SuppressWarnings`